main/src/main/java/cgeo/geocaching/CacheDetailActivity.java

```diff
@@ -224,6 +226,11 @@ public class CacheDetailActivity extends TabbedViewPagerActivity
 
     private MenuItem menuItemToggleWaypointsFromNote = null;
 
+    private CompassMiniView compassMiniView;
+    private float azimuth = 0;
+
+    private final boolean isLiveDetail = Settings.isLiveDetail();
```

Comment:
Don't cache settings in member variables. Users might change it while the activity is still alive. 
If you need to access it often, add a method "getXYYZ"  and use it.